command/server.go

```diff
@@ -2589,6 +2589,7 @@ func setSeal(c *ServerCommand, config *server.Config, infoKeys []string, info ma
 
 		var wrapperInfoKeys []string
 		wrapperInfoMap := map[string]string{}
+		configSeal.Config = make(map[string]string)
```

Comment:
I think you need a `nil` check here. Otherwise, won't this end up overwriting values provided in the config file?